lib/cli/src/Cardano/CLI.hs

```diff
@@ -1217,57 +1228,65 @@ instance FromText TxId where
 
 -- | Controls how much information to include in log output.
 data Verbosity
-    = Default
-        -- ^ The default level of verbosity.
-    | Quiet
+    = Quiet
         -- ^ Include less information in the log output.
     | Verbose
         -- ^ Include more information in the log output.
     deriving (Eq, Show)
 
--- | Convert a given 'Verbosity' level into a list of command line arguments
---   that can be passed through to a sub-process.
-verbosityToArgs :: Verbosity -> [String]
-verbosityToArgs = \case
-    Default -> []
-    Quiet   -> ["--quiet"]
-    Verbose -> ["--verbose"]
-
 -- | Map a given 'Verbosity' level onto a 'Severity' level.
 verbosityToMinSeverity :: Verbosity -> Severity
 verbosityToMinSeverity = \case
-    Default -> Info
-    Quiet   -> Error
     Verbose -> Debug
+    Quiet -> Error
+
+allTracers :: [Text]
+allTracers = map ("cardano-wallet" <>)
+    [ ".api"
+    , ".daedalus-ipc"
+    , ".database"
+    , ".database.query"
+    , ".stake-pools"
+    , ".worker"
+    , "" ]
```

Comment:
I suppose this last one works like a wildcard? I haven't played with it but I wonder how we can pass in an empty tracer like this? `--quiet` or `--verbose` "naked" (with no argument) ?